### DISCUSSION TITLE
Github added Mathematics (MathJax) to Markdown can npmjs.com do the same?

### DISCUSSION THREAD

**Discussion by jacobbogers** [npm/feedback#855] [Website] [answered]
Hello,

Github recently added mathematic symbols to their markdown README.md files (github uses MathJax)

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#what

I see this is not possible in npmjs.com README.md, is this on the todo list to be added?

Cheers
Jacob Bogers

**Comment by MylesBorins** [comment, marked as answer]
Hey @jacobbogers. We are not currently prioritizing work related to markdown rendering on the site. At this time our roadmap is entirely focused on improvements related to Account and Package Security such as Granular Access Tokens and Package Provenance
